packages/waku/src/server.ts

```diff
@@ -167,11 +170,13 @@ type PlatformObject = {
       | 'buildClientBundle'
       | 'buildDeploy';
   };
-} & Record<string, unknown>;
+} & T;
 
 (globalThis as any).__WAKU_PLATFORM_OBJECT__ ||= {};
 
 // TODO tentative name
-export function unstable_getPlatformObject(): PlatformObject {
+export function unstable_getPlatformObject<
+  T = Record<string, unknown>,
+>(): PlatformObject<T> {
```

Comment:
I believe asserting types is more appropriate than annotating types, because it never be type safe.